crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -163,10 +179,82 @@ impl From<Network> for clap::builder::OsStr {
     }
 }
 
+fn parse_cors(inputs: Vec<String>) -> Result<Option<AllowedOrigins>, RpcCorsDomainsParseError> {
```

Comment:
nit: using `Vec<&str>` could save a lot of `.clone()` calls in tests.

Reply:
Unfortunately clap produces owned strings and trying to use `Vec<impl AsRef<str>>` is insufficient and blew up in my face.  Moreover one of the error variants requires an owned string anyway.